src/widget/OktaSignIn.js

```diff
@@ -95,7 +95,7 @@ var OktaSignIn = (function () {
   function OktaSignIn (options) {
     require('okta');
 
-    var OktaAuth = require('@okta/okta-auth-js/jquery');
+    var OktaAuth = require('@okta/okta-auth-js');
```

Comment:
**Note:** New version of okta-auth-js uses `cross-fetch` by default! (https://github.com/okta/okta-auth-js/pull/199), which is we want to use cross-fetch in version 3.0. So we don't need to specify `/jquery` version anymore.

(This has been already cross-browser + embedded browser tested for the widget when we made the change in okta-auth-js).